release.nix

```diff
@@ -0,0 +1,4 @@
+let
+  pkgs = import <nixpkgs> { };
```

Comment:
Consider doing this via a function argument as well, like in `shell.nix` — this allows overriding the version of nixpkgs used from the outside.